packages/website/src/components/layout/EditorTabs.tsx

```diff
@@ -36,6 +38,13 @@ function EditorTabs<T extends boolean | string>({
             onClick={(): void => change(item.value)}
           >
             {item.label}
+            {additionalTabsInfo?.[item.label] ? (
+              <div>
+                {additionalTabsInfo[item.label] > 9
+                  ? '9+'
```

Comment:
"9+" seems really low. Does it significantly impact layout? If not can we use "99+"?

Reply:
I adjusted the layout to make longer letters look clear, and then modified it to show up to 99+!

so layout is like below images.

<img width="95" alt="스크린샷 2024-03-12 오후 8 56 35" src="https://github.com/typescript-eslint/typescript-eslint/assets/102564722/2622808b-79bc-4b22-b8c3-6f2233b4cda7">

<img width="95" alt="스크린샷 2024-03-12 오후 8 56 49" src="https://github.com/typescript-eslint/typescript-eslint/assets/102564722/2359f034-7618-496f-a38f-78fd492ca7ee">

<img width="108" alt="스크린샷 2024-03-12 오후 8 56 58" src="https://github.com/typescript-eslint/typescript-eslint/assets/102564722/f5afead3-70cf-4052-9380-d6a3095230dd">